sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EChangeFeedSpec.scala

```diff
@@ -15,6 +15,10 @@ class SparkE2EChangeFeedSpec
   //scalastyle:off multiple.string.literals
   //scalastyle:off magic.number
 
+  override def afterEach(): Unit = {
+    this.reinitializeContainer()
```

Comment:
I think you can use the `AutoCleanableCosmosContainer` trait to achieve the same thing.

Reply:
No - AutoCleanable deletes all records vs. deleting the container

Reply:
Which for Incremental Changefeed doesn't have same effect